firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteSchema.java

```diff
@@ -50,6 +50,14 @@
   // Remove this constant and increment VERSION to enable indexing support
   static final int INDEXING_SUPPORT_VERSION = VERSION + 1;
 
+  /**
+   * The batch size for operations that potentially operate on a large result set.
+   *
+   * <p>This addresses https://github.com/firebase/firebase-android-sdk/issues/370, where a customer
+   * reported that schema migrations failed for clients with thousands of documents.
+   */
+  static final int REMOTE_DOCUMENTS_BATCH_SIZE = 100;
```

Comment:
Is there some reason 100 is guaranteed to be safe?  From the code / comments I can't tell if it was chosen arbitrarily or if there's a specific rationale behind it.

Reply:
This number is arbitrary. I picked a smaller number to make the unit tests run faster. I bumped it to 1000, which I believe to be safe as well since I have only ever hit this condition with 10k+ documents.

I will verify this change in another manual test run.

Reply:
Test run succeeded with 1000.